src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -138,7 +138,8 @@ def determineOutputModules(self, scenarioFunc=None, scenarioArgs=None,
                                       'SingleTrack': 'AOD',
                                       'MinBias': 'AOD',
                                       'OniaUPC': 'RAW-RECO',
-                                      'HighMET': 'RECO'
+                                      'HighMET': 'RECO',
+                                      'BPHSkim': 'USER'
                                      }
                     dataTier = skimToDataTier.get(physicsSkim, 'USER')
```

Comment:
Trick works but looks a little ugly IMO. Plus do we really care if a patch changes one or two lines ?

Same sentiment for the default to tier USER. In principle correct, but I still like to keep a list of all know physics skims in one place. Also, most physics skims don't have data tier USER and I don't want to get into the habit of "we can just add the new physics skim to the config without adding it to the code". That default is to catch mistakes, not as something intentionally used.